src/content/configuration/index.md

```diff
@@ -396,9 +383,23 @@ module.exports = {
 }
 ```
 
-T> Want to rapidly generate webpack configuration file for your project requirements with few clicks away.
+## Use custom config file
+
+If for some reason you want to use custom config file depending on certain situations you can change this via command line by using the `--config` flag.
+
+__package.json__
+
+```json
+"scripts": {
+  "build": "webpack --config prod.config.js"
+}
+```
+
+## Config generators
```

Comment:
maybe `config` -> `configuration` in this title sits well in left hand side menu?